gen.py

```diff
@@ -136,19 +137,14 @@ def convert_svg2png(infile, outfile, w, h):
         for size in sizes:
             root = "{0}.svg".format(icon["linux"]["root"])
             source = "icons/{0}/{1}/{2}.svg".format(theme, size, icon_name)
-            output = "{0}/{1}/apps/{2}".format(linux_dir, size, root)
+            output = "{0}/{1}/apps/".format(linux_dir, size)
             if path.exists(source):
-                bfb_icon = icon["linux"].get("bfb")
-                if bfb_icon:
-                    output_bfb = "{0}/{1}/apps/{2}.png".format(linux_dir,
-                                                               size,
-                                                               bfb_icon)
-                    if int(size) == 48:
-                        convert_svg2png(source, output_bfb, 144, 144)
-                copy2(source, output)
+                if icon["linux"].get("bfb") and (size == "48"):
+                    output_bfb = "{0}.png".format(icon["linux"].get("bfb"))
+                    convert_svg2png(source, output+output_bfb, 144, 144)
+                copy2(source, output+root)
```

Comment:
Same here

Reply:
There's no need for parenthesis in `size == "48"`

Reply:
My thinking was that
```python
# Option 1
if gbfb and (size == "48"):
    print("Do the thing")
print("Do the next thing")
```
is clearer than just
```python
# Option 2
if gbfb and size == "48":
    print("Do the thing")
print("Do the next thing")
```
because it makes the two sides of the `and` clear. This also works because logic:
```python
# Option 3
if (gbfb and size) == "48":
    print("Do the thing")
print("Do the next thing")
```